src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -307,14 +308,40 @@ extension DynamicTableViewController {
 		if dynamicTableViewModel.section(at: indexPath).isHidden(for: self) {
 			return UITableViewCell()
 		}
-
+		
 		let section = dynamicTableViewModel.section(at: indexPath)
 		let content = dynamicTableViewModel.cell(at: indexPath)
 
 		let cell = tableView.dequeueReusableCell(withIdentifier: content.cellReuseIdentifier, for: indexPath)
+		
+		
+		cell.removeBackground()
+		
+		switch section.background {
+		case .none:
+			cell.backgroundColor = .clear
+		case .greyBoxed:
+			
+			// Give all subviews some insets to the border
+			if let subview = cell.subviews.first?.subviews.first {
```

Comment:
Some magic code here. Which view is getting the inset here? Is it the content view?

Reply:
It's about the views inside the content view. We cannot change the insets of the content view. I added a comment for better understanding.